controller/userController.js

```diff
@@ -60,17 +60,17 @@ const updateUser = async (req, res) => {
     res.send(result).status(200);
 }
 
-async function getUserNameByAddress(userAdress) {
+async function getUserByAddress(userAdress) {
     const db = dbo.getDb();
     let collection = await db.collection("users");
     let query = {_id: userAdress};
     let result = await collection.findOne(query);
-    return (result.name)
+    return (result)
```

Comment:
What are the attributes of the result

Reply:
{
        "_id": "0x9B0365F4C6F086A871a5F2d4e2e86D290936Cd5B",
        "name": "Chess Lords",
        "symbol": "CLS",
        "tokenType": "ERC4907",
        "createdBy": "0xBe1f9dae05301e114cA10342903415361b9cD1E7",
        "createdAt": "2023-02-18T08:36:36.205Z",
        "modifiedAt": "2023-02-18T08:36:36.205Z",
        "createdUserName": "Isuru97",
        "createdUserImage": "https://res.cloudinary.com/isuruieee/image/upload/v1676639701/00073_ysx5m1.png",
        "count": 7,
        "tokens": [
            "https://gateway.pinata.cloud/ipfs/QmXeCLbf6BFeWxfL67jzQ7PNJWEUfostbEfY1octUvwq6u",
            "https://gateway.pinata.cloud/ipfs/QmcqUveJ9auzbF24yyDb34QKcZvJCjt6tCMMELs2Y2gaH9",
            "https://gateway.pinata.cloud/ipfs/QmRj8McEoERoPrtCtSJoYxfndW1jMgrJz3p3wH19eyT6nR",
            "https://gateway.pinata.cloud/ipfs/QmXWsVv1xGBPZPEZScB8keE4bbBVpHzLaMq8aDPHa28F4M"
        ]
    }